annex/annexA.md

```diff
@@ -0,0 +1,19 @@
+ANNEX
+-----
+
+This annex is the follow-up of examples related to chapter 1 in this tutorial. 
+
+The example7.cpp is about: 
+- Listener to print the content of the event and the thread id
+- Publisher and subscriber for other types of data: int, my own struct such as Position(float _x, float _y)
```

Comment:
Remove pronouns "my" as it is not clear who "me" is.  Make the text more generic and impersonal.

Reply:
Done